File: go/Baekjun/04.array/10818.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	var reader *bufio.Reader = bufio.NewReader(os.Stdin)
	var writer *bufio.Writer = bufio.NewWriter(os.Stdout)

	defer writer.Flush()

	var N int

	fmt.Fscanln(reader, &N)

	var init int
	fmt.Fscan(reader, &init)
	var min, max int = init, init

	for i := 1; i < N; i++ {
		var input int
		fmt.Fscan(reader, &input)

		if input < min {
			min = input
		}

		if input > max {
			max = input
		}
	}

	fmt.Fprintln(writer, min, max)

}

File: C/Baekjoon/10818.c
#include <stdio.h>

int main() {
  int N;
  scanf("%d", &N);
  
  int max= -1000000, min = 1000000;

  for(int i=0; i < N; i++) {
    int inputData;
    scanf("%d", &inputData);
    if(inputData > max) max = inputData;
    if(inputData < min) min = inputData;
  }

  printf("%d %d", min, max);
  
  return 0;
}